LambdaRuntimeDockerfiles/dotnet5/Dockerfile

```diff
@@ -54,9 +54,9 @@ RUN curl -SL --output aspnetcore.tar.gz https://dotnetcli.azureedge.net/dotnet/a
 
 
 FROM base as final
-ARG DOTNET_VERSION
+ARG ASPNET_VERSION
```

Comment:
Does it automatically take the value assigned here https://github.com/aws/aws-lambda-dotnet/blob/master/LambdaRuntimeDockerfiles/dotnet5/Dockerfile#L3 ?

Reply:
Yes, I built the image and tested that.